frontend/src/pages/Monitor/index.jsx

```diff
@@ -1,94 +1,150 @@
+import { useEffect, useState } from "react";
 import { PageTitle, MemorySvg } from "../../components";
-import { MdCreate, MdArrowBack, MdMonitor } from "react-icons/md";
-import { useLocation } from "react-router";
+import { MdCreate, MdArrowBack, MdMonitor,MdCheck } from "react-icons/md";
+import { useLocation, useParams } from "react-router";
+import monitorService from "../../services/monitorService";
+import monitorsGroupService from "../../services/monitorsGroupService";
 
 
 function Monitor(){
     const { state } = useLocation();
-    const monitor = state;
+    const [monitor,setMonitor] = useState(state);
+    const [name,setName] = useState("");
+    const [groupId,setGroupId] = useState();
+    const [groups,setGroups] = useState([]);
+    const { id } = useParams();
+    const [Update,setUpdate] = useState(false);
```

Comment:
This does not match other state styles.
And if this is an `updater` then please name it that to keep it consistent throughout the files.